App/Stylesheets/index.css

```diff
@@ -1,16 +1,14 @@
-:root {
-    --body-bg:  rgb(192, 210, 228);
-    --primary-bg: rgb(224, 240, 255);
-    --shadow-color:  rgb(152, 205, 255);
-    --mobileDesc-bg: rgb(192, 226, 228);
-}
```

Comment:
Why are these 6 lines deleted, if you use the `--primary-bg` variable below in `line 27`?